src/EFCore.PG/Infrastructure/Internal/NpgsqlOptionsExtension.cs

```diff
@@ -195,6 +195,12 @@ public virtual NpgsqlOptionsExtension WithRemoteCertificateValidationCallback([C
             return clone;
         }
 
+        public override void PopulateDebugInfo(IDictionary<string, string> debugInfo)
+        {
+            // We migth want to add the current values of AdminDatabase, PostgresVersion, ReverseNullOrdering, _userRangeDefinitions
```

Comment:
Yeah, in theory I think we need to include everything that's configurable on the options extension.

nit: typo (`migth`)

Reply:
Ok.
I think that implementing this will be my next commit.
Question: Should we populate debugInfo only if the values have been set to something different than their default value or should we put in default values too?

Reply:
I'm not sure, but we should follow whatever the EF Core providers do. I seem to remember that they include information even if it's the default - we should confirm that.

Reply:
The [SQL Server provider](https://github.com/aspnet/EntityFrameworkCore/blob/025a5589b25d4b96318d4ba2d4e9aed0d32e527d/src/EFCore.SqlServer/Infrastructure/Internal/SqlServerOptionsExtension.cs#L88) and the [Sqlite provider](https://github.com/aspnet/EntityFrameworkCore/blob/ca6f9a8ef8aa4bea9e8cb085cf5e5bd21cf84ba7/src/EFCore.Sqlite.Core/Infrastructure/Internal/SqliteOptionsExtension.cs#L85) both log unconditionally. On the other hand they have few to no configurable options to log anyways.

Reply:
Good point. I have no idea what the effect would be of always adding all the options, but I'm guessing it could make logging much heavier. It may make sense to only add info when it deviates from the default.